src/inputgroup/skin.css

```diff
@@ -86,3 +102,79 @@
     }
   }
 }
+
+.spectrum-Datepicker--range {
+  .spectrum-InputGroup-field {
+    &:focus-ring {
+      box-shadow: none !important;
+    }
+    &[disabled] {
+      ~ hr {
```

Comment:
Can we give the `<hr>` a classname and target that?

Reply:
I there shouldn't be an `<hr>`, just `.spectrum-Datepicker--range-dash`.  I think this is a holdover from an earlier version.